services/web-app/components/dashboard/dashboard.module.scss

```diff
@@ -29,3 +29,99 @@
     padding-left: $spacing-05;
   }
 }
+
+.title {
+  padding-bottom: $spacing-02;
+  color: $text-secondary;
+  @include type-style('body-short-01');
+}
+
+.subtitle {
+  padding-bottom: $spacing-07;
+  color: $text-primary;
+  @include type-style('body-short-02');
+}
+
+.maintainer {
```

Comment:
Can the class names in the Dashboard component be generic? Not specific to the content? For example, what happens when we use the dashboard in a standards page? Instead of "maintainer" class name to make the font size bigger, it could be some "large" or "big" modifier to a "value" or generic class for that copy.